cg/cli/downsample.py

```diff
@@ -0,0 +1,74 @@
+"""cg module for downsampling reads in a sample."""
+
+import logging
+from pathlib import Path
+from typing import Tuple
+
+import click
+
+from cg.apps.downsample.downsample import DownsampleAPI
+from cg.apps.downsample.utils import store_downsampled_sample_bundle
+from cg.constants.constants import DRY_RUN
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def downsample():
+    """Downsample reads in a sample."""
+
+
+@downsample.command("samples", help="Downsample reads in one or multiple samples in a case.")
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="Case identifier used in statusdb, e.g. supersonicturtle. The case information wil be transferred.",
+)
+@click.option(
+    "-i",
+    "--input-data",
+    required=True,
+    nargs=2,
+    multiple=True,
+    help="Identifier used in statusdb, e.g. ACC1234567 and the number of reads to down sample to in millions separated by a space"
+    " e.g. ACC1234567 30. Multiple inputs can be provided.",
```

Comment:
I do not think it is clear here that the user can supply `0.001` as a possible value for the number of reads.